icons/font-size-lower.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="m5 18 4-8 4 8m-7-2h6M14 6l2 2 2-2" />

```

Comment:
What about something like this? ![baseline (1)](https://user-images.githubusercontent.com/25524993/236631667-b5e9e51e-0fa8-44ae-b544-0d152d0b9b01.svg)
```suggestion
  <path d="m4 18 6-12 6 12" />
  <path d="M5.5 15h9" />
  <path d="M15.5 6 18 8.5 20.5 6" />
```

Reply:
I don't mind, I think that the original looks a bit misaligned compared to yours so that looks good